srcs/SetServer_utils.cpp

```diff
@@ -8,24 +8,32 @@ void Server::closeClient(int i, int fd, Client *client)
     std::map<std::string, Client *>	clientlist;
     bool found = false;
 
-	close(fd);
     std::map<std::string, Channel*>::iterator it;
     for(it=_channels.begin(); it!=_channels.end(); it++)
     {
         clientlist = it->second->getClientList();
-        std::map<std::string, Client*>::iterator it2;
-        for (it2=clientlist.begin(); it2!=clientlist.end(); it2++)
+        if (clientlist.size() == 0)
```

Comment:
I dont think this situetion should be possible? if the channel is removed when tehre is 0 clients in it?

Reply:
nope it shouldn't but for some reason, we were keep getting fsanitize address problem there, so I just made a check that we won't go through it if it is empty, and at least I didn't get the random fsanitize problem.